src/libraries/Microsoft.Extensions.Options/src/StartupValidatorOptions.cs

```diff
@@ -9,6 +9,6 @@ namespace Microsoft.Extensions.Options
     internal sealed class StartupValidatorOptions
     {
         // Maps each pair of a) options type and b) options name to a method that forces its evaluation, e.g. IOptionsMonitor<TOptions>.Get(name)
-        public Dictionary<(Type, string), Action> _validators { get; } = new Dictionary<(Type, string), Action>();
+        public Dictionary<(Type, string), Action> _validators { get; } = new();
```

Comment:
This is a style-only change and should not be mixed with other changes. We generally do not accept style-only change case by case without corresponding analyzer rule.